src/Map.js

```diff
@@ -72,7 +90,7 @@ const initialState = {
   meetups: {},
   attestations: {}
 };
-
+// THIS USE-CASES trigger the action to save to the state calles 'state' , which is done by the setter 'dispatch'
```

Comment:
Ah, i actually wanted to add some descriptions, so it would be clearer for someone else (and for me if i have to get back to it. I'll try to phrase it clearer)